share/spice/pirate_speak/pirate_speak.css

```diff
@@ -0,0 +1,14 @@
+.zci__body h3 {
```

Comment:
@thejasbabu let's remove this CSS -- We have other IA's that modify/translate and we don't use the quotes there. So far we're only using them when displaying an actual quote.